include/solvers/navier_stokes_scratch_data.h

```diff
@@ -188,6 +190,14 @@ class NavierStokesScratchData : public PhysicsScratchDataBase
     if (sd.gather_mortar)
       enable_mortar();
 
+    if (sd.gather_particle_field_project)
+      enable_particle_field_projection(
+        sd.fe_values_particle_drag->get_quadrature(),
+        sd.fe_values_particle_drag->get_mapping(),
+        sd.fe_values_particle_drag->get_fe(),
+        sd.fe_values_particle_two_way_coupling_force->get_fe(),
+        sd.fe_values_particle_velocity->get_fe());
```

Comment:
The quadrature and the mappig should be those used for the Navier-Stokes equation (so the velocity and pressure), not necessarily the one of the drag. I understand they will be the same by construction, but still.